package.json

```
@@ -2,13 +2,17 @@
   "private": true,
   "scripts": {
     "dev": "astro dev",
-    "build": "astro build"
+    "build": "astro build",
+    "format": "prettier -w src astro.config.ts"
```

I think it's better to format all files since Prettier isn't just for JavaScript
```suggestion
    "format": "prettier -w ."
```